### ISSUE TITLE
Defining Code of Conduct for Stories

### ISSUE THREAD

**Issue by skfantoni** [alan-turing-institute/AutSPACEs#282]
## Summary

Working out what is appropriate to post in a story.

## Date Due by

TBC

## Details

* Agree on what counts as ableist or offensive re autism
* Decide what counts as inappropriate re race, genre, sexuality etc.
* Agree on how much of the story should be specifically about sensory issues
* Decide how specific people can be in re to places and people

## Suggestions for tasks that can fix this issue

* [ ] Gather and collate existing comments and ideas relating to this issue
* [ ] Discuss this issue with group at meet-up
* [ ] Create a red-list of words/terms re autism
* [ ] Create a red-list of words/terms re other discrimination 
* [ ] Write up first draft CoC for Stories
* [ ] Review decisions with a larger group
* [ ] Finalise CoC for Stories

## Who can help

Anyone interested in the moderation process

**Comment by GeorgiaHCA** [comment]
Fabulous work on this everyone! Closing this issue now as we've now written the Code of conduct
